ios/AirMaps/AIRMap.m

```diff
@@ -67,6 +73,23 @@ - (instancetype)init
         // be identical to the built-in callout view (which has a private API)
         self.calloutView = [SMCalloutView platformCalloutView];
         self.calloutView.delegate = self;
+        
+        self.cacheImageView = [[UIImageView alloc] initWithFrame:self.bounds];
```

Comment:
Is instantiating all of these views (imageview + view + indicatorview) potentially a bit wasteful if these props aren't there?